doc/en/source/ngsildbasediotintegration.rst

```diff
@@ -0,0 +1,476 @@
+			"value": ""
+		}
+	}
+	
+The **observedAt** shows last the time any command associated with the entity has been invoked. The result of **on** command can be seen in the value of the **on_info** attribute.
+
+How to Fetch data from Orion-LD to FogFlow 
+================================================================
+
+
+**Step5** Issue a subscription to Orion-LD broker. 
+-------------------------------------------------------------------
+
+.. code-block:: console    
+
+	curl -iX POST \
+		  'http://localhost:1026/ngsi-ld/v1/subscriptions' \
+		  -H 'Content-Type: application/json' \
+		  -H 'Accept: application/ld+json' \
+		  -H 'fiware-service: openiot' \
+		  -H 'fiware-servicepath: /' \
+		  -H 'Link: <https://fiware.github.io/data-models/context.jsonld>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
+		  -d ' {
+                 	"type": "Subscription",
+                	"entities": [{
+				"id": "urn:ngsi-ld:Device:water001",
+                               "type": "Device"
+                 	}],
+             	      "notification": {
+                          "format": "normalized",
+                          "endpoint": {
+                                   "uri": "http://<fogflow_broker_IP>:8070/ngsi-ld/v1/notifyContext/",
+                                   "accept": "application/ld+json"
+             	           }
+                       }
+ 	           }'
+
+**Step6** send the below PATCH request to Enable Orion-Broker commands
+-------------------------------------------------------------------
+
+.. code-block:: console 
+
+	curl -L -X PATCH 'http://localhost:1026/ngsi-ld/v1/entities/urn:ngsi-ld:Device:water001/attrs/on' \
+	-H 'fiware-service: openiot' \
+	-H 'fiware-servicepath: /' \
+	-H 'Accept: application/ld+json' \
+	-H 'Link: <https://fiware.github.io/data-models/context.jsonld>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
+	-H 'Content-Type: application/json' \
+	--data-raw '{
+
+        	"type": "Property",
+        	"value": " "
+
+	}'
+	
+**Step7** Check if FogFlow receives the subscribed entity. 
+-------------------------------------------------------------------
+
+Use the CURL command to query entities of type "Device" from  FogFlow thinBroker. 
+
+
+.. code-block:: console    
+
+	curl -iX GET \
+		  'http://localhost:8070/ngsi-ld/v1/entities?type=Device' \
+		  -H 'Content-Type: application/json' \
+		  -H 'Accept: application/ld+json' \
+		  -H 'fiware-service: openiot' \
+		  -H 'fiware-servicepath: /' \
+		  -H 'Link: <https://fiware.github.io/data-models/context.jsonld>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' 
+
+Note: Replace localhost with the IP where fogflow thinbroker is running.
+Note: Replace the localhost with IP where Orion-LD broker is running and <fogflow_broker_IP> with the IP where the fogflow broker is running.
+
+
```

Comment:
correct the image operator registration. "fogflow/waterpump" should be there

Reply:
Done